src/panels/map/ha-panel-map.js

```diff
@@ -129,7 +133,7 @@ class HaPanelMap extends LocalizeMixin(PolymerElement) {
           className: "",
```

Comment:
Use the `className` here too.

Reply:
Implemented in https://github.com/home-assistant/home-assistant-polymer/pull/4085/commits/9d10ab8f054f4dcf50bc10b0768c8194b4359c2b